scene/gui/line_edit.cpp

```diff
@@ -2280,7 +2276,14 @@ void LineEdit::_shape() {
 	if (text.length() == 0 && ime_text.length() == 0) {
 		t = placeholder_translated;
 	} else if (pass) {
-		t = secret_character.repeat(text.length() + ime_text.length());
+		// Todo: Integrate with text server to add support for non-latin scripts.
```

Comment:
```suggestion
		// TODO: Integrate with text server to add support for non-latin scripts.
```
Will pop in IDEs and easier to spot